api/src/test/java/org/openmrs/notification/db/hibernate/HibernateAlertDAOTest.java

```diff
@@ -32,11 +47,50 @@ public void runBeforeEachTest() {
 	 * @throws Exception
 	 */
 	@Test
-	public void shouldGetAlerts() {
-		
-		AlertService as = Context.getAlertService();
-		//System.out.println(as.getAllAlerts());
+	public void getAlert_shouldReturnSavedAlert() throws Exception {
```

Comment:
Did you forgot to add @test annotation here, you can also try to indent your code, use indentation

Reply:
@sherrif10   got fixed thanks for the quick check up

Reply:
No @HerbertYiga  you havent fixed **getAlert_shouldReturnSvedAlert**,  Private Alert createAlert is missing test annotation if at all is a test, thanks for this

Reply:
You can feel free to first run your tests, get acquited on how to use datasets where its necessary they give alot of experience